book/website/community-handbook/bookdash/bookdash-application.md

```diff
@@ -0,0 +1,95 @@
+(ch-bookdash-application)=
+# Application and Review Process
+
```

Comment:
Given what is written below on rubrics for assessment, would it be worth including examples of the other questions applicants have to answer alongside accessibility and diversity info? If you would prefer not to disclose it that's fine too but might be worth adding a line for context.

Reply:
I think it is a good idea. We do have the application form public so this can be written transparently here.

Reply:
Added questions from the application: https://github.com/alan-turing-institute/the-turing-way/pull/1262/commits/9d1acb2a88132f353358b3878c6ed3881ce562bf